src/scripts.js

```diff
@@ -52,6 +61,41 @@ function sortRecipesByName() {
   recipeRepository.recipes.sort((a, b) => a.name - b.name);
 }
 
+function displayRecipe() {
+  const card = event.target.parentNode;
```

Comment:
Where is `event` coming from?

Reply:
Added that event parameter to the function declaration. :)